sormas-e2e-tests/src/test/resources/features/sanity/web/Case.feature

```diff
@@ -9,6 +9,17 @@ Feature: Case end to end tests
     Then I check the created data is correctly displayed on Edit case page
     And I check the created data is correctly displayed on Edit case person page
 
+  Scenario:Check that double clicking NEW CASE button does not cause a redundant action
```

Comment:
now that I see the steps got me a bit confused with the aim of this test.
It's a bit useless since the click on new case button cannot be performed once we have the case creation popup in view.
I'll start a discussion thread on the jira ticket to understand the aim.

Reply:
I would add another check here. Maybe is not added in the test case, but most of the tests are navigating also to other tabs and collect data from there as well. So when the Immunization POJO is created, it has also the name, last name, sex etc, and that data isn't checked because exists in the Person tab (from Immunization view).
Now, we can stick to individual test case automation, but this will mean to keep in mind and come back and update this test once we'll start working on that check